### PR TITLE
DDCNL-2674

### PR DESCRIPTION
**Pull request by oc-ste**

### CODE REVIEW

File: conf/messages.cy [hunk 1]
```diff
@@ -168,53 +168,66 @@ eothoQuestions.title = Rhoi adborth - GOV.UK
...
+eothoQuestions.encourageReopenSooner.heading = A wnaeth y cynllun eich annog i ailagor yn gynt, neu i ailagor am fwy o ddiwrnodau’r wythnos, nag oeddech wedi’i fwriadu?
+eothoQuestions.encourageReturnToRestaurantsSooner.heading = A ydych o’r farn bod y cynllun wedi annog pobl i ddychwelyd i fwytai yn gynt nag y byddent wedi fel arall?
+
+numberOfEstablishmentsQuestion.OneEstablishment = 1
+numberOfEstablishmentsQuestion.TwoToTenEstablishments = 2 i 10
+numberOfEstablishmentsQuestion.TenOrMoreEstablishments = 10 neu fwy
+numberOfEstablishmentsQuestion.NationalChainEstablishment = Cadwyn genedlaethol
+
+numberOfEmployeesQuestion.NoEmployees = Dim
+numberOfEmployeesQuestion.OneToTenEmployees = -	1 i 10
```
Comment (danewatts) [inline conf/messages.cy]: Is this dash and spacing supposed to be here?

File: conf/messages.en [hunk 2]
```diff
@@ -157,52 +157,65 @@ eothoQuestions.title = Give feedback - GOV.UK
...
+eothoQuestions.encourageReturnToRestaurantsSooner.heading = Do you think the scheme encouraged people to return to restaurants sooner than they would have otherwise?
+
+numberOfEstablishmentsQuestion.OneEstablishment = 1
+numberOfEstablishmentsQuestion.TwoToTenEstablishments = 2 to 10
+numberOfEstablishmentsQuestion.TenOrMoreEstablishments = 10 or more
+numberOfEstablishmentsQuestion.NationalChainEstablishment = National chain
+
+numberOfEmployeesQuestion.NoEmployees = None
+numberOfEmployeesQuestion.OneToTenEmployees = 1 to 10
+numberOfEmployeesQuestion.ElevenToHundredEmployees = 11-100
```
Comment (danewatts) [inline conf/messages.en:183]: Was this a design choice to have one answer X to Y and the other answers X-Y?

File: conf/messages.cy [hunk 3]
```diff
@@ -168,53 +168,66 @@ eothoQuestions.title = Rhoi adborth - GOV.UK
...
+comparedBusinessTurnover.IncreasedByMore50percent = Cyfanswm trosiant wedi cynyddu mwy na 50%
 
-offerDiscounts.NoDoNotPlanToOfferDiscountsInSeptember = Nac ydy, nid ydym yn bwriadu cynnig gostyngiadau ym mis Medi
-offerDiscounts.YesOfferUpTo50pcOnMonTueWedInSeptember = Ydy, hyd at 50% oddi ar fwyd a diod a brynir i’w fwyta ar y safle ar ddydd Llun, dydd Mawrth a dydd Mercher ym mis Medi
-offerDiscounts.YesOffer50pcOrMoreOnMonTueWedInSeptember = Ydy, 50% neu fwy oddi ar fwyd a diod a brynir i’w fwyta ar y safle ar ddydd Llun, dydd Mawrth a dydd Mercher ym mis Medi
-offerDiscounts.YesOfferDifferentDiscountsOrDealsInSeptember = Ydy, rydym yn bwriadu cynnig gostyngiadau neu fargeinion gwahanol ym mis Medi
+offerDiscounts.NoDoNotPlanToOfferDiscountsInSeptember = Nac ydy - nid ydym yn bwriadu cynnig gostyngiadau ym mis Medi
+offerDiscounts.YesOfferUpTo50pcOnMonTueWedInSeptember = Ydy - hyd at 50% oddi ar fwyd a diod a brynir i’w fwyta ar y safle ar ddydd Llun, dydd Mawrth a dydd Mercher ym mis Medi
+offerDiscounts.YesOfferDifferentDiscountsOrDealsInSeptember = Ydy - rydym yn bwriadu cynnig gostyngiadau neu fargeinion gwahanol ym mis Medi
```
Comment (danewatts) [inline conf/messages.cy:230]: The Welsh and English seem to diverge with this change. Is this expected?

File: conf/messages.cy [hunk 4]
```diff
@@ -265,7 +278,7 @@ whichRegions.heading = Ym mha ranbarthau mae sefydliadau’ch busnes? Ticiwch bo
 whichRegions.EastMidlands = Dwyrain Canolbarth Lloegr
 whichRegions.EastOfEngland = Dwyrain Lloegr
 whichRegions.London = Llundain
-whichRegions.NorthEast = Gogledd-ddwyrain Lloegr
+whichRegions.NorthEast = ogledd-ddwyrain Lloegr
```
Comment (danewatts) [inline conf/messages.cy]: This change seems to say "the north east" as opposed to the other options which just say "north east". Is this expected?

**Review by danewatts — CHANGES_REQUESTED** [review]

File: conf/messages.cy (hunk 1 above)
Comment (danewatts) [reply conf/messages.cy]: This is now resolved in latest commit.

**Comment by danewatts** [comment]
Also as discussed in slack, wrong quote in message file, different hyphens used for Yes - and No - between welsh and english messages and the change to english message moving from "and" to "&" causes a diverge in welsh content

File: conf/messages.en (hunk 2 above)
Comment (oc-ste) [reply conf/messages.en:183]: Did not design this, just being given this from chancellor team

File: conf/messages.cy (hunk 3 above)
Comment (oc-ste) [reply conf/messages.cy:230]: Gone with hyphens here instead

File: conf/messages.cy (hunk 4 above)
Comment (oc-ste) [reply conf/messages.cy]: Fixed, had a missing G

**Review by danewatts — APPROVED** [review]

**Pull request merged**